client/src/pages/spectrogram/components/annotation/AnnotationViewer.tsx

```diff
@@ -185,7 +185,7 @@ const AnnotationViewer = ({
 
   return (
     <Layer>
-      {/* Button to add a new annotation */}
```

Comment:
Presuming you didn't mean to remove the Button here

Reply:
Good spot! I did not

Reply:
Added that back in :)